src/main/java/ru/practicum/shareit/item/ItemMapper.java

```diff
@@ -18,4 +30,23 @@ public Item fromDto(ItemDto itemDto, User owner) {
         return Item.builder().id(itemDto.getId()).name(itemDto.getName()).description(itemDto.getDescription())
                 .available(itemDto.getAvailable()).owner(owner).build();
     }
+
+    public OwnerItemDto toDtoOwner(Item item, Booking lastBooking, Booking nextBooking, List<Comment> commentsList) {
+        List<CommentDto> comments = commentsList.stream().map(
+                this::toCommentDto).collect(Collectors.toList());
+        OwnerItemDto itemDto = OwnerItemDto.builder().id(item.getId()).name(item.getName()).comments(comments)
+                .description(item.getDescription()).available(item.getAvailable()).build();
+        if (lastBooking != null) {
+            itemDto.setLastBooking(bookingMapper.getShortDto(lastBooking));
+        }
+        if (nextBooking != null) {
+            itemDto.setNextBooking(bookingMapper.getShortDto(nextBooking));
+        }
+        return itemDto;
+    }
+
+    public CommentDto toCommentDto(Comment c) {
+        return CommentDto.builder().id(c.getId()).text(c.getText()).authorName(c.getAuthor().getName())
```

Comment:
Для лучшей читаемости в коммерческой разработке принято бить билдеры по строкам:
```suggestion
        return CommentDto.builder()
            .id(c.getId())
            .text(c.getText())
            .authorName(c.getAuthor().getName())
```